### ISSUE TITLE
Bug: Pasting plain text into RichTextPlugin does not preserve line breaks

### ISSUE THREAD

**Issue by chrysb** [facebook/lexical#5592]
Lexical version: 0.13.1

## Steps To Reproduce

1. Copy text from a normal textarea that has line multiple line breaks
2. Paste into Lexical with a RichTextPlugin

```
    <LexicalComposer
      key='editor'
      initialConfig={{
        ...editorConfig,
        editorState: defaultValue,
      }}
    >
      <RichTextPlugin
        contentEditable={
          <ContentEditable className='outline-none w-full leading-7 h-full dark:text-gray-300 text-gray-800' />
        }
        placeholder={<Placeholder />}
        ErrorBoundary={LexicalErrorBoundary}
      />
      <HistoryPlugin />
      <ListPlugin />
      <LinkPlugin />
      <ListMaxIndentLevelPlugin maxDepth={2} />
      <MarkdownShortcutPlugin transformers={TRANSFORMERS} />
      <AutoLinkPlugin />
    </LexicalComposer>
```

## The current behavior

```
Sentence once. Sentence two.
```

## The expected behavior
```
Sentence one.

Sentence two.
```

**Comment by Liakhov** [comment]
I have the same issue.

And also have an issue, during pasting HTML, for example:

**const htmlData = dataTransfer.getData('text/html');**
**htmlData:**
`<meta charset='utf-8'><div>first paragraph</div><div>first paragraph</div>`. - in this case we will have 1 paragraph without line breaks.

The correct line breaks I can get only if `htmlData` consists of  `p` and `span` tags for the HTML.

**Comment by lukasb** [comment]
I just reproduced this bug with 0.14.2. Updated to 0.16.0 and it went away.

**Comment by jnnwnk** [comment]
I've got a similar issue. I copy text from google docs (which results in data transfer of type `text/html`). The HTML looks like this (I removed styling attributes for better overview):

```html
<html>
<body>
    <!--StartFragment-->
    <meta charset="utf-8">
    <b>
    	<br />
        <p dir="ltr">Test1</span><span><br /></span><span>Test2</span></p>
        <p dir="ltr">Test3</span></p>
    </b><br class="Apple-interchange-newline">
    <!--EndFragment-->
</body>
</html>
```

The first line break (before `p` tag) is recognized correctly.
The second line break (which is embedded between Test1 and Test2 in the first paragraph) gets lost.

Any idea how to fix this? I just updated to version 0.16.0 but the issue still exists.
I would als try to help working on a fix if someone points out how/where to start.

I tried to debug the code already. What I figured out so far: This snippet in `LexicalHtml.dev.mjs` returns null because `isOnlyChild()` returns `true`. But I'm not sure if this is how to solve it.
```ts
static importDOM() {
    return {
      br: node => {
        if (isOnlyChild(node)) {
          return null;
        }
        return {
          conversion: convertLineBreakElement,
          priority: 0
        };
      }
    };
  }
```

Any help appreciated.
